agg-ast/schema_derivation/src/schema_derivation.rs

```diff
@@ -1111,6 +1211,56 @@ impl DeriveSchema for TaggedOperator {
                 }
                 Ok(Schema::Array(Box::new(array_schema)))
             }
```

Comment:
This is unfortunate, I tried to use a macro, but you can't use macros for pattern expansions, sadly.